src/apps/api/tests/test_competitions.py

```diff
@@ -183,56 +188,33 @@ def test_get_competition_leaderboard_as_json(self):
-        # Makes sure the query returns a json that had a matching leaderboard title
-        leaderboard_choice = random.choice(list(self.leaderboard_ids_to_titles.values()))
-        url = reverse('competition-results', kwargs={"pk": self.comp.id})[0:-1] + f'.json?title={leaderboard_choice}'
-        response = self.client.get(url, HTTP_ACCEPT='application/json')
-        self.assertEqual(response.status_code, 200)
-        content = json.loads(response.content)
+                response_users.add(user)
+            print(f'{key} : {self.usernames} == {response_users}')
+            assert self.usernames == response_users
 
-        for title in content.keys():
-            assert leaderboard_choice in title
+        response_title = str(list(self.response_titles)[0]).split(' ')[0]
+        assert self.leaderboard.title in response_title
 
     def test_get_competition_leaderboard_by_id_as_json(self):
         # Make sure when getting leaderboard by id you get exactly one leaderboard with matching title
-        leaderboard_choice = random.choice(list(self.leaderboard_ids_to_titles.keys()))
-        url = reverse('competition-results', kwargs={"pk": self.comp.id})[0:-1] + f'.json?id={leaderboard_choice}'
+        phase_choice = random.choice(self.phases)
+        url = reverse('competition-results', kwargs={"pk": self.comp.id})[0:-1] + f'.json?phase={phase_choice.id}'
         response = self.client.get(url, HTTP_ACCEPT='application/json')
         self.assertEqual(response.status_code, 200)
         content = json.loads(response.content)
 
         response_title = list(content.keys())
         assert len(response_title) == 1
-        assert response_title[0] == f'{self.leaderboard_ids_to_titles[leaderboard_choice]}({leaderboard_choice})'
+        assert response_title[0] == f'{self.leaderboard.title} - {phase_choice.name}({phase_choice.id})'
 
     def test_get_competition_leaderboard_by_id_as_csv(self):
-        leaderboard_choice = random.choice(list(self.leaderboard_ids_to_titles.keys()))
-        url = reverse('competition-results', kwargs={"pk": self.comp.id})[0:-1] + f'.csv?id={leaderboard_choice}'
-        response = self.client.get(url, HTTP_ACCEPT='text/csv')
-        self.assertEqual(response.status_code, 200)
-
-        content = response.content.decode('utf-8')
-        csv_reader = csv.reader(StringIO(content))
-        csv_header = list(csv_reader)[0]
-        csv_header.pop(0)
-
-        for column_title in self.leaderboard_ids_to_columns[leaderboard_choice]:
-            assert f'{column_title}({self.leaderboard_ids_to_columns[leaderboard_choice][column_title]})'
-
-    def test_get_competition_leaderboard_by_title_as_csv(self):
-        leaderboard_choice = random.choice(list(self.leaderboard_ids_to_titles.keys()))
-        url = reverse('competition-results', kwargs={"pk": self.comp.id})[0:-1] + f'.csv?title={self.leaderboard_ids_to_titles[leaderboard_choice]}({leaderboard_choice})'
```

Comment:
Same thing here. It squashed 2 methods together after I deleted "test_get_competition_leaderboard_by_title_as_csv".